app/controllers/admin/event_schedules_controller.rb

```diff
@@ -31,5 +31,11 @@ def destroy
     def event_schedule_params
       params.require(:event_schedule).permit(:schedule_id, :event_id, :room_id, :start_time)
     end
+
+    def parse_errors(event_schedule)
+      title = event_schedule.event.try(:title).present? ? event_schedule.event.title : 'The event'
```

Comment:
Why / How can that happen?
event_schedule validates presence of event and event validates presence of title, so this case shouldn't happen, should it?